.azure-pipelines/client.yml

```diff
@@ -0,0 +1,62 @@
+trigger:
+- master
+
+jobs:
+- job: 'Build_Target_Packages'
+
+  pool:
+    vmImage: 'ubuntu-16.04'  
+
+  steps:
+  - task: UsePythonVersion@0
+    displayName: 'Use Python 3.6'
+    inputs:
+      versionSpec: 3.6
+
+  - script: |
+     python -m venv .\
+     
+     pip install git+https://github.com/lmazuel/swagger-to-sdk
```

Comment:
You do not need this if you don't build drops

Reply:
Makes perfect sense. I'll rip it out.